.github/ISSUE_TEMPLATE/pre-release.md

```diff
@@ -0,0 +1,98 @@
+---
+name: Pre-Release
+title: [vX.XX] Pre-Release Procedure
+assignees: @floreks @maciaszczykm @kgroschoff
+about: The list of steps that should be executed prior to the release step list 
+labels: sig/ui sig/release
+---
+
+## Pre-Release Procedure
+List of steps to execute before proceeding with the [release procedure](docs/manuals/release.md#release-procedure). It mainly consists of
+manual tests and visual checks of the Kubermatic Dashboard.
```

Comment:
Should we also test project, service accounts & members (add/edit/delete) + display all projects as an admin?
Or is this something we rely on the e2e tests?

Reply:
> I mean we can't test the provider stuff without a project, but just to complete the list...?

We've dropped projects from manual testing for exactly this reason. There is no need to remember about that as it will always be tested anyway. As for the SA & Members, we've got e2e tests for them that already check basically everything that is possible to check there. There is no need to do this manually then. Wizard & clusters are much more complex and e2e do not always catch everything there.

Reply:
Okay understand. 👍 
But still - maybe a quick check if the `display all projects` function is working? I think we don't test it somewhere else, do we? 🤔

Reply:
Good idea with the `display all projects` switch. We do not have a test for that. I'll add a check.

Reply:
Done